Frends.Community.Email/Definitions.cs

```diff
@@ -189,6 +262,18 @@ public enum SecureSocketOption
         StartTlsWhenAvailable
     }
 
+    public enum AuthenticationMethod
```

Comment:
I could not find the basic auth usage or for that matter, I do not think that this enum is used anywhere at all.